definitions/ext-site/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: EXT
+type: SITE
+
+synthesis:
+    # telemetry with SiteID attribute
+    name: SiteID
+    identifier: SiteID
+    encodeIdentifierInGUID: true
+    conditions:
+    - attribute: SiteID
+
+dashboardTemplates:
+  newRelic:
+    template: dashboard.json
+
+configuration:
+  entityExpirationTime: QUARTERLY
```

Comment:
Hi @mlowe9999,

last question, then we would be good to go ahead and merge from my point of view.
Could we lower this duration - e.g. to a day ?
Or is there a specific use case for retaining those entities for a long time ?

Asking because every long-lived entity type impacts the platform much more storage-wise.
Thank you.